tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -136,6 +149,19 @@ ZTEST_F(ascs_test_suite, test_sink_ase_read_state_idle)
 	zassert_equal(0x00, hdr.ase_state, "unexpected ASE_State 0x%02x", hdr.ase_state);
 }
 
+ZTEST_F(ascs_test_suite, test_cb_register_without_ascs_registered)
```

Comment:
Suggest to also test attemptso double register as well as double unregister.

If possible, also unregister when there's an ASE in a non-idle state. 

Ideally we would test all return values and cases